### ISSUE TITLE
Using Google Sans font

### ISSUE THREAD

**Issue by romaluca** [flutter/flutter#18266]
How can i include Google sans font in my app like in flutter gallery app?
Where i can download it?
Thanks

**Comment by dhayaec** [comment]
refer this guide to use custom fonts
https://flutter.io/cookbook/design/fonts/

I found the fonts in this repo
https://github.com/psingl/product-sans-font

**Comment by willlarche** [comment]
@dhayaec , I would not use Product Sans either. It isn't licensed for the public either.

**Comment by CarsonRedeye** [comment]
This is quite a shame, that google is forcing inconsistency of fonts between apps on android devices

**Comment by MarcinOrlowski** [comment]
No, it is not a shame. What is a shame is attitude like yours . Not everything in the world comes for free because you think you deserve it. You do not and noone is obliged to sponsor you. You may don't like that but they most likely paid for that and if the want to keep it for themselves then all you can do is accept that.

**Comment by CarsonRedeye** [comment]
@MarcinOrlowski What are you talking about? The shame is that google is forcing inconsistency in the operating system, which is bad design. I don't care about cost, I care about the android ecosystem.

**Comment by MarcinOrlowski** [comment]
You arguments are pretty rubbish. Google apps are NOT android. Android is AOSP. No version of Android uses Google Sans. Android uses Roboto. So there's no inconsistency.

**Comment by willlarche** [comment]
I __love__ the passion that you two are showing for design, Android, and Google's fonts. I think you're both right in a lot of respects! Let me explain.

There's a lot of super successful products that faithfully implemented the original look of Material Design. (The one they found both in our guidelines and our Android apps.) And we've been blown away at how dedicated the community is to consistency and 'doing it right.'

Over time we realized a great opportunity for improvement on our part was in helping apps differentiate themselves from each other. We updated the guidelines to include some tools for doing this. See [Material Theming](https://material.io/design/material-theming/overview.html). The main idea is choosing colors, fonts, shapes, etc that express _your_ brand so users really see, and feel, your app is one-of-a-kind and quintesentially **you**.

The first company we tried it on was us! Our [typography](https://material.io/design/typography/the-type-system.html#applying-the-type-scale) theme includes Google Sans as a header font and Roboto for body. Google Sans shares a lot of traits with our logo like circular Gs. It's a quintesentially Google font.

We know it's not possible for most apps to have custom fonts so we suggest you look around [Google Fonts](https://fonts.google.com/) to pick one from the thousands and thousands of free-to-use fonts you'll find there. But if you can't decide, Roboto is still great! It's a font we've spent a lot of time on and it's made to be read on screens.

And if you don't agree, that's also ok. Feedback like that is really important for us to hear. In fact, it sounds like we could improve how we explain this so we avoid misunderstandings in the future.

But for now, let's set aside this acrimony :) The Flutter community is the kindest community I've ever interacted with. Excitement for the framework is greater than our feelings about fonts. I promise. 

You can learn more in our [video](https://www.youtube.com/watch?v=hA0hrpR-o8U) on the story and use of Material Theming in Flutter.

**Comment by JavierSegoviaCordoba** [comment]
@MarcinOrlowski I think Google is using it in the system (Android Q). If it is in the system, I think they should be possible to use in all apps.

![image](https://user-images.githubusercontent.com/7463564/57698577-35991280-7656-11e9-94c0-bb3d89f8d485.png)

**Comment by MarcinOrlowski** [comment]
It should. On Q.

**Comment by pkitatta** [comment]
How about PT Sans? It is a font I was using for my ionic app, which app I am trying to migrate to flutter. The reason am on this page is that am trying to use the same font for the new version. I hope I have been using PT Sans legally.

**Comment by JaimeChavarriaga** [comment]
@pkitatta [PT Sans](https://fonts.google.com/specimen/PT+Sans), [Roboto](https://fonts.google.com/specimen/Roboto) and the other fonts in [Google Fonts](https://fonts.google.com/) are free to use. Please note that the license information is shown in the corresponding Google Font page. For instance, PT Sans uses the [Open Font License](http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL_web) and Roboto uses [the Apache License](http://www.apache.org/licenses/LICENSE-2.0). 

Google Sans or Product Sans are not fonts licensed to developers.
